htlcswitch/service.go

```diff
@@ -0,0 +1,73 @@
+package htlcswitch
+
+import (
+	"github.com/go-errors/errors"
+	"sync"
+	"sync/atomic"
+)
+
+// Service...
+type Service struct {
```

Comment:
This file looks incomplete. In any case, I don't see a direct purpose for this struct. The prior "service" like methods on structs were typically unique to the particualr sub-system with distinct setup/tear down logic. I don't think this struct improves the code much and seems to be a bit over-abstracted.